notebooks/official/ray_on_vertex_ai/spark_on_ray_on_vertex_ai.ipynb

```diff
@@ -0,0 +1,1353 @@
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "53100b3cbe10"
+      },
+      "source": [
+        "[Ray on Vertex AI container images](https://cloud.google.com/vertex-ai/docs/supported-frameworks-list#ray) do not come with RayDP pre-installed, Create a custom Ray on Vertex AI container image to run Spark applications on Ray on Vertex AI. The following section explains how a custom container image for Ray on Vertex AI with RayDP can be built."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "01a3fc372b9e"
+      },
+      "source": [
+        "Create a directory to store the dockerfile"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "2c25577425e3"
+      },
+      "outputs": [],
+      "source": [
+        "DOCKER_DIR = \"docker_dir\""
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "3c482e0c7b1c"
+      },
+      "outputs": [],
+      "source": [
+        "!mkdir -p {DOCKER_DIR}"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "b0a46bce3de8"
+      },
+      "source": [
+        "Recommended to use the latest Ray on Vertex AI prebuilt image for creating the custom container image. Install other Python packages that are expected to be used by the Spark applications. \n",
+        "\n",
+        "Note: pyarrow==14.0 is due to a dependency constraint of Ray 2.9.3 and it also fails to read csv with pandas version >= 2.2.0."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "3468ec9a1f3c"
+      },
+      "outputs": [],
+      "source": [
+        "%%writefile {DOCKER_DIR}/Dockerfile\n",
+        "\n",
+        "FROM us-docker.pkg.dev/vertex-ai/training/ray-cpu.2-9.py310:latest\n",
+        "\n",
+        "RUN apt-get update -y \\\n",
+        "    && pip install --no-cache-dir raydp pyarrow==14.0 pandas==2.1.0"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "a28c2e599e48"
+      },
+      "source": [
+        "### Enable Artifact Registry API\n",
+        "Enable the Artifact Registry API service for the Google cloud project. This tutorial assumes that [gcloud CLI](https://cloud.google.com/sdk/docs/install) is already installed."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "26b48ef4ba29"
+      },
+      "outputs": [],
+      "source": [
+        "! gcloud services enable artifactregistry.googleapis.com"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "b37f718f7eae"
+      },
+      "source": [
+        "### Create a private Docker repository\n",
+        "Create a Docker repository in Google Artifact Registry."
```

Comment:
- remove "Google"
- add link to the Artifact Registry overview page:
    - [Artifact Registry](https://cloud.google.com/artifact-registry/docs/overview)